src/EA.Weee.Web/Areas/Admin/ViewModels/Organisation/EditRegisteredCompanyOrganisationDetailsViewModel.cs

```diff
@@ -12,8 +12,8 @@ public class EditRegisteredCompanyOrganisationDetailsViewModel : EditOrganisatio
         public string CompanyName { get; set; }
 
         [Required(ErrorMessage = "Enter company registration number (CRN)")]
-        [StringLength(maximumLength: EnvironmentAgencyMaxFieldLengths.CompanyRegistrationNumber, MinimumLength = 7, ErrorMessage = "The Company registration number should be 7 to 15 characters long")]
-        [Display(Name = "Company registration number (CRN)")]
+        [StringLength(maximumLength: EnvironmentAgencyMaxFieldLengths.CompanyRegistrationNumber, MinimumLength = 7, ErrorMessage = "The company registration number should be 7 to 15 characters long")]
+        [Display(Name = "company registration number (CRN)")]
```

Comment:
Don't think this one should be changed, the one in the sentence looks ok.

Reply:
restored